include/generators/order_polytope_generator.h

```diff
@@ -38,14 +53,32 @@ static const std::unordered_map<std::string, std::string> instances =
 
 };
 
+// generates a Polytope from a poset
+/// @tparam Polytope Type of returned polytope
+template <class Polytope>
+Polytope get_orderpoly(Poset const &poset) {
+    typedef typename Polytope::PointType Point;
+
+    OrderPolytope<Point> OP(poset);
+    if constexpr (std::is_same< Polytope, OrderPolytope<Point> >::value ) {
+        return OP;
+    } else if constexpr (std::is_same<Polytope, HPolytope<Point> >::value ){
+        Polytope HP(OP.dimension(), OP.get_full_mat(), OP.get_vec());
+        return HP;
+    } else {
+        // TODO: implement functionality for more polytope types
```

Comment:
A side note here: creating a V-polytope from an H-polytope representation of an order polytope is a hard problem (and cannot be approximated as volume computation) thus it is realistic to just remove that TODO ;)

Reply:
I don't know how useful it would be, but I feel like in this special case of order polytope created from a poset, it might not be as hard, but maybe I'm wrong